modules/ldapbackend/testfiles/add.ldif

```diff
@@ -1,24 +1,21 @@
-dn: olcDatabase=hdb,cn=config
+dn: olcDatabase=mdb,cn=config
 objectClass: olcDatabaseConfig
-objectClass: olcHdbConfig
-olcDatabase: hdb
+objectClass: olcMdbConfig
+olcDatabase: mdb
 olcDbDirectory: /var/lib/ldap/powerdns
 olcSuffix: o=power
 olcLastMod: TRUE
 olcRootDN: uid=testuser,o=power
 olcDbCheckpoint: 512 30
-olcDbConfig: {0}set_cachesize 0 2097152 0
-olcDbConfig: {1}set_lk_max_objects 1500
-olcDbConfig: {2}set_lk_max_locks 1500
-olcDbConfig: {3}set_lk_max_lockers 1500
 olcDbIndex: objectClass eq
 olcDbIndex: associatedDomain eq
 olcRootPW: secret
+OlcDbMaxSize: 2684354560
```

Comment:
random magic number, the default wasn't big enough. probably tunable to something smaller.